dashboard/config/linux/bits/debug.yml

```diff
@@ -21,7 +21,7 @@ config:
  - CSD_LOCK_WAIT_DEBUG: [v5.19, -arm]
  - DETECT_HUNG_TASK
  - WQ_WATCHDOG
- - BOOTPARAM_HUNG_TASK_PANIC
+ - BOOTPARAM_HUNG_TASK_PANIC: [1, linux-next]
```

Comment:
Note that it will only set it on `linux-next`, otherwise it will stay the default value.